scripts/travis_ci/travis_script.sh

```diff
@@ -25,19 +25,28 @@ if [ "$RUN_ARM_BUILD" = "true" ]; then
 fi
 
 if [ "$RUN_X86_TESTS" = "true" ]; then
-    BUILD_DIR=boards/x86_build
-    # Use Release build for testing because the heartbeat monitor relies on it
-    # for code logic
-    travis_run cmake -S boards -B $BUILD_DIR -DPLATFORM=x86 -DCMAKE_BUILD_TYPE=Release
-    travis_run make --directory=$BUILD_DIR
-    travis_run "cd $BUILD_DIR && ctest --verbose"
+    # The heartbeat monitor logic is different in Release and Debug build type,
```

Comment:
Not so relevant to this PR but since I was cleaning up the test files I thought I'd fix this too